### ISSUE TITLE
Add func for admin to manually add another part to an existing order

### ISSUE THREAD

**Issue by andwaredev-zz** [BroceMfg/broce-client#55]
Add func for admin to manually add another part to an existing order.

When we spoke with the parts guys out in Dodge City, they told us it's fairly common for a customer to call them after placing an order and ask to add another part(s) to that order. They want this to remain the way it happens (rather than giving clients the option to edit an order again after it has been created (although we will want to give clients the option to cancel an order see issue #56 )).

Basically, the scenario would be that a customer places an order and a couple hours or maybe even a day later they call to ask if the parts guys can also through another part in with the order.

The parts guys want to have manual control over whether or not this part can in fact get added to the existing order or if needs to become a new order entirely.

This means we'll want to add this "add another part" func to the admin's frontend view.
